api/definitions.yaml

```diff
@@ -97,8 +97,31 @@ NewMessage:
       $ref: "#/MessageContent"
     default_addresses:
       $ref: "#/NewMessageDefaultAddresses"
+    payment_data:
```

Comment:
I would put `payment_data` inside `MessageContent`, they have the same lifecycle